CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 * Add [CheckboxChoicesWidget](https://prestojs.com/docs/ui-antd/widgets/CheckboxChoicesWidget/)
 * Add [ChoicesWidget](https://prestojs.com/docs/ui-antd/widgets/ChoicesWidget/)
 * Add [parseTime](https://prestojs.com/docs/util/parseTime/) and [formatTime](https://prestojs.com/docs/util/formatTime/)
+* Add [path](https://prestojs.com/docs/rest/paginationMiddleware/) option to `paginationMiddleware`. This allows pagination to be handled on a sub-key of the data.
```

Comment:
I don't love the name... Maybe resultKey ?

Reply:
key sounds like it's only a direct key and not a dotted path... `resultPath`?

Reply:
Hmm... maybe a very explicit `dottedResultKey`/`dottedResultPath`/`dottedResultProperty`?

Reply:
Also, result or response?

Reply:
it can be dotted or not though (and most cases will be not)

`result` because that's what we call it after the body has been decoded from response and transformed

Reply:
Go with `resultPath` then.